app/common/UserAPI.ts

```diff
@@ -481,6 +482,11 @@ interface SqlResult extends TableRecordValuesWithoutIds {
   statement: string;
 }
 
+export const DocAttachmentsLocation = StringUnion(
+  "NO FILES", "INTERNAL", "MIXED", "EXTERNAL"
```

Comment:
If you look at other string unions in this codebase I think they lean camelCase. Also maybe just `none` instead of `noFiles`.

Reply:
You're absolutely right - I'll swap the casing around here :)